tests/install_swiftpm_dependencies/test_scripts/test_project_automatic.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -eu
+
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE[0]}")/set_up_environment.sh"
+
+echo "--- :computer: Jump to test folder"
+pushd "$TESTS_LOCATION/../project"
+
+brew install xcodegen
+make
+# Also resolve packages to generate Package.resolved in expected location
+xcodebuild -resolvePackageDependencies -project Demo.xcodeproj
+
+echo "--- :wrench: Run install_swiftpm_dependencies"
+install_swiftpm_dependencies
```

Comment:
Doesn't it defeat the purpose of the test to run `xcodebuild -resolvePackageDependencies` **before** running `install_swiftpm_dependencies`? 🤔 💭

Reply:
Ok I now realize that the goal is to validate that `install_swiftpm+dependencies` uses the rght `Package.resolved` from the right path based on the `--flags` passed to it (or lack thereof), so maybe that makes sense after all…

But still, this feels like this initial `xcodebuild -resolvePackageDependencies -project Demo.xcodeproj` could easily generate a false positive on the integration test.

What do you think about instead just keep a `Fixture-Package.resolved` file in the `project/` folder, and replace this `xcodebuild -resolvePackageDependencies -project Demo.xcodeproj` with a `cp Fixture-Package.resolved Demo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved` so that we still have a Package.resolved but without invoking `xcodebuild` before calling the `install_swiftpm_dependencies` SUT and risking side effects?

Granted, that kinda violates what you mention in the `README` about testing Apple tools as integration tests, as we'd instead assume the path where those are generated by `xcodebuild`. But I think that's still a valid assumption to make and unlikely to change, and I'd be ok to do that if that allows us to avoid the risk of side effects and test false positives.

Besides, if we go the route of using `cp` to put the `Package.resolved` in place:
 - That will make the test more stable by using a fixture for the `Package.resolved`, while calling `xcodebuild -resolvePackageDependencies` to let Xcode resolve the dependencies and create it will potentially create a different `.resolved` file at different times (depending on the latest versions of the dependencies available online at the time), and thus make for harder to reproduce test failures (can't easily `git bisect` to try to repro failures on an old build for example)
 - That will allow use to also `cp` an `InvalidPackageFixture.resolved`(one that would fail compiling, could even just be a file with sole content being `let package = invalid`) in the `Demo.xcodeproj`, while copying the valid `PackageFixture.resolved` into the `Demo.xcworkspace/project.xcodeproj/…`, which would allow us to validate that `install_swiftpm_dependencies` would use the right one when testing it on the `workspace_and_project` integration test below…